scripts/translation-status.ts

```diff
@@ -92,6 +103,15 @@ class TranslationStatusBuilder {
 		output.debug('');
 	}
 
+	/** Get all pull requests with the `i18n` tag */
+	async getPullRequests() {
+		const pullRequests = (await fetch(
+			`https://api.github.com/repos/${this.githubRepo}/pulls?state=open&per_page=50`
+		).then((res) => res.json())) as PullRequest[];
```

Comment:
Pointing out we have a function for getting data from GitHub, you can find it in [scripts/lib/github-get.mjs](https://github.com/withastro/docs/blob/main/scripts/lib/github-get.mjs). It has retry, headers and makes easier to add our GitHub token to avoid the request being limited, could you make the changes to use it in here? Here's how we're using it in the [generate-docgen-integration-pages.ts](https://github.com/withastro/docs/blob/main/scripts/generate-integration-pages.ts#L73) script if that helps!

Also heads up that adding an env variable possibly will block the PR/deploy for security reasons, if this happens we will take action to unblock it, so no need to worry! 🙌

Reply:
That's a handy util. Thanks for linking the usage too! That made it a lot quicker to add :smile: